dashboard/app/views/admin_reports/level_answers.csv.erb

```diff
@@ -6,5 +6,5 @@ CSV.generate do |csv|
       csv << response
     end
   end
-end
+end .html_safe
```

Comment:
Hm, why do you need html_safe? Since all you are doing here is hacking around erb I think it may make sense to do this more like (in the controller):

csv = CSV.generate {....}

render plain: csv, content_type: 'text/csv'